src/App.test.jsx

```diff
@@ -0,0 +1,19 @@
+// test list displays
+import { render, screen, waitForElementToBeRemoved } from '@testing-library/react'
+import App from './App'
+
+describe('App', () => {
+  it('shoulder render a list of characters', async () => {
+    // render App
+    render(<App />)
+
+    // loading
+    screen.getByAltText(/avatar wheel of punishment spinner/i);
+
+    await waitForElementToBeRemoved(screen.getByAltText(/avatar wheel of punishment spinner/i));
+
+    // find character
+    const result = await screen.findByText('Sokka');
+    expect(result.textContent).toEqual('Sokka');
```

Comment:
While this test passes, it doesn't really verify that all of your page is displaying properly. There are three objects in your mocked data, so I would expect that all of them are present on the screen.  You can `findAllBy` and then verify that the text content of each item in the array `[0],[1], and [2]` are present. Just some things to think about when you are testing a component in a unit test like this.